cluster/gce/util.sh

```diff
@@ -2738,20 +2738,22 @@ function create-master() {
   declare -a APISERVER_RESERVED_IP
   declare -a APISERVER_RESERVED_INTERNAL_IP
   declare -a APISERVER_NAME
-  CREATE_CERT_APISERVER_IP=""
+
+  # add all external and internal IP to cert
+  CREATE_CERT_SERVER_IP="${MASTER_RESERVED_INTERNAL_IP}"
```

Comment:
Why rename CREATE_CERT_APISERVER_IP to CREATE_CERT_SERVER_IP?

Reply:
previous, this var only used for additional apiserver, but now, it is used for all servers IP including master internal ip, additional apiserver internal ip and external ip. and also may include extra ETCD cluster ip in the future, so rename to server_IP